system/views/renderers/content/auditLogEntry/default.cfm

```diff
@@ -0,0 +1,21 @@
+<cfparam name="args" type="struct"/>
+<cfoutput>
+	<table class="table formbuilder-response table-striped">
+		<tr>
+			<th>#translateResource( "cms:auditTrail.action" )#</th>
+			<td>#args.action#</td>
+		</tr>
+		<tr>
+			<th>#translateResource( "cms:auditTrail.type" )#</th>
+			<td>#args.type#</td>
+		</tr>
+		<tr>
+			<th>#translateResource( "cms:auditTrail.detail" )#</th>
+			<td>#args.detail#</td>
+		</tr>
+		<tr>
+			<th>#translateResource( "cms:auditTrail.Date&Time" )#</th>
```

Comment:
Please make the i18n property name `auditTrail.dateAndTime` (do not use non alpha numeric characters in the names, other than `.`).